Client/Frontend/Login Management/BreachAlertsClient.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  BreachAlertsClient.swift
+//  Client
+//
+//  Created by Vanna Phong on 5/27/20.
+//  Copyright © 2020 Mozilla. All rights reserved.
+//
+
+import Foundation
+import Shared
+
+/// Errors related to BreachAlertsClient and BreachAlertsManager.
+struct BreachAlertsError: MaybeErrorType {
+    public let description: String
+    public init(description: String) {
+        self.description = description
+    }
+}
+/// For mocking and testing BreachAlertsClient.
+protocol BreachAlertsClientProtocol {
+    func fetchData(endpoint: BreachAlertsClient.Endpoint, completion: @escaping (_ result: Maybe<Data>) -> Void)
+}
+
+/// Handles all network requests for BreachAlertsManager.
+public class BreachAlertsClient: BreachAlertsClientProtocol {
+    private var dataTask: URLSessionDataTask?
+    public enum Endpoint: String {
+        case breachedAccounts = "https://monitor.firefox.com/hibp/breaches"
+    }
+
+    public init() { }
```

Comment:
This init can be removed as well since it doesn't take any parameters

Reply:
The compiler was upset with me for a bit, saying `BreachAlertsClient` was internal, so I defined a public init and it was happy. Fiddling final/public left this init behind, and it compiles fine without this public init now so I'll remove it